includes/discovery/sensors/count/projector.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @subpackage webui
+ * @link       http://librenms.org
+ * @copyright  2018 LibreNMS
+ * @author     LibreNMS Contributors
+*/
+
+use Illuminate\Support\Str;
+
+if (Str::startsWith($device['sysObjectID'], '.1.3.6.1.4.1.1248.4.1')) {
```

Comment:
This should be possible to implement in yaml, would you mind giving it a try?

Reply:
Is this possible without mibs? Haven't found some so I did this workaround.

Reply:
Hmm I think you can just use the oids

Reply:
OK, sure then I will try it of course

Reply:
`
modules:
    sensors:
        count:
            data:
                -
                    num_oid: '.1.3.6.1.4.1.1248.4.1.1.1.1.{{ $index }}'
                    descr: Lamp Hours
		    index: 0
`
Does nor work :-(

Reply:
Need to add Oid also I guess

Reply:
unfortunately also does not work with Numeric OIDs

Reply:
the sensors module does not support numeric only in yaml :/